core/driver/sources/src/main/resources/reference.conf

```diff
@@ -5,12 +5,52 @@ probe {
 
   // `intellij` is for org.virtuslab.ideprobe.config.IntellijConfig - default values below:
   intellij {
+    // `plugins` array contains IntelliJ plugins which should be installed and enabled on IntelliJ run. Here you can add
+    // your IntelliJ plugin for tests (amongst other plugins). There are a few ways to provide a proper entry pointing
+    // to a plugin:
+    // 1. By using a direct URI pointing to the plugin (could be from a website or from a local file). For example:
+    #     { uri = "https://packages.jetbrains.team/maven/p/ij/intellij-dependencies/com/intellij/remoterobot/robot-server-plugin/0.11.15/robot-server-plugin/0.11.15.zip" }
+    // 2. By using default IntelliJ plugins repository (see `probe.resolvers.plugins.repository.uri`) with additional
```

Comment:
If `probe.resolvers.plugins.repository.uri` is **changed from the default**, then the is _default IntelliJ plugins repository_ or the newly overridden repository used for resolving the plugins? (I suppose the latter, but the use of _default_ in this sentence suggests the former)

Reply:
The value from `probe.resolvers.plugins.repository.uri` will be used as the _default IntelliJ plugins repository_ to resolve the Plugin to a valid URL. Let me make it more clear.